nightguard WatchKit App/NotificationExtension.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  NotificationExtension.swift
+//  nightguard WatchKit App
+//
+//  Created by Philipp Pöml on 29.05.24.
+//  Copyright © 2024 private. All rights reserved.
```

Comment:
This should be removed, since it clashes with the nightscout license.

Reply:
Sure, this is something Xcode buts into every file, like the whole project is full of it:
```
//
//  ViewExtension.swift
//  nightguard
//
//  Created by Dirk Hermanns on 08.04.23.
//  Copyright © 2023 private. All rights reserved.
//
```
But you can put your name, if that is better.

Reply:
Your name here is fine. Helps. But simply delete the copyright line.

I will remove it in all other files.